src/azure/cli/utils/_msrestazure_threading.py

```diff
@@ -0,0 +1,24 @@
+#---------------------------------------------------------------------------------------------
```

Comment:
Let’s create a pull request making the default to be a daemon thread the default for msrestazure.

From: Travis Prescott <notifications@github.com>
Reply-To: Azure/azure-cli <reply@reply.github.com>
Date: Thursday, August 25, 2016 at 2:25 PM
To: Azure/azure-cli <azure-cli@noreply.github.com>
Cc: Johan Stenberg <Johan.Stenberg@microsoft.com>, Assign <assign@noreply.github.com>
Subject: Re: [Azure/azure-cli] Exit From LongRunningOperations with Ctrl+C (#757)


In src/azure/cli/utils/_msrestazure_threading.py<https://github.com/Azure/azure-cli/pull/757#discussion_r76327792>:

> @@ -0,0 +1,24 @@

> +#---------------------------------------------------------------------------------------------

Yes. I created an issue in their repo: Azure/autorest#1378<https://github.com/Azure/autorest/issues/1378>

The current implementation does not expose the ability to create daemon threads. This would work around the issue until such time that they (maybe) make this the default or configurable.

Another solution would be great too, but just trying to stop the running threads on KeyboardInterrupt failed.

________________________________

In reply to: 76327054<https://github.com/Azure/azure-cli/pull/757#discussion_r76327054>

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub<https://github.com/Azure/azure-cli/pull/757/files/0f8a313c13a82d0e5fd2a41159146426e70d6191#r76327792>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AOaPgtXBnymItyZDJekY7zr5D2oxDr05ks5qjghYgaJpZM4JtgtP>.